File: Lodjinha/app/src/main/java/br/com/bsavoini/lodjinha/api/model/ProductsResponse.java
package br.com.bsavoini.lodjinha.api.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class ProductsResponse {

    @SerializedName("data")
    private List<ProductModel> productsArr;

    @SerializedName("offset")
    private Integer offset;

    @SerializedName("total")
    private Integer total;

    public List<ProductModel> getProductsArr() {
        return productsArr;
    }

    public Integer getOffset() {
        return offset;
    }

    public Integer getTotal() {
        return total;
    }
}

File: data/src/main/java/br/com/cemobile/data/remote/models/ProductsResponse.kt
package br.com.cemobile.data.remote.models

import br.com.cemobile.domain.model.Product
import com.google.gson.annotations.SerializedName

data class ProductsResponse(
    @SerializedName("data") val data: List<Product>,
    @SerializedName("offset") val offset: Int,
    @SerializedName("total") val total: Int
)